### PR TITLE
Fix Typing For Vertices.mean - Returns single vector, instead of array<vector>

### PR DESCRIPTION
**Pull request by BalticBytes**
Please fill in this template.

- [y] Use a meaningful title for the pull request. Include the name of the package modified.
- [-] Test the change in your own code. (Compile and run.)
- [-] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [y] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [y] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [y] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [y] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/liabru/matter-js/blob/826ed46829ced96b7fb310e219b24a62afe46712/src/geometry/Vertices.js#L105

### CODE REVIEW

**Comment by BalticBytes** [comment]
Hey, hope everything is according to your standard!

TL;DR: The function in question in the current version of [`matter-js`](https://github.com/liabru/matter-js)
```    /**
     * Returns the average (mean) of the set of vertices.
     * @method mean
     * @param {vertices} vertices
     * @return {vector} The average point
     */
    Vertices.mean = function(vertices) {
        var average = { x: 0, y: 0 };

        for (var i = 0; i < vertices.length; i++) {
            average.x += vertices[i].x;
            average.y += vertices[i].y;
        }

        return Vector.div(average, vertices.length);
    };
```

**Review by hasparus — APPROVED** [review]
Huge thanks for the PR!
